frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -60,32 +60,31 @@ function HeaderText(text: string) {
     )
 }
 
-function UsedAndRemainingTime(mission: Mission) {
+function StartUsedAndRemainingTime(mission: Mission) {
+    var startTime
     var remainingTime
-    let usedTime = 0
+    var usedTime
 
     let dateTime = mission.estimatedDuration.split('.')
     const days = dateTime.length === 1 ? 0 : dateTime[0].split(':')[0]
     const time = dateTime.length === 1 ? dateTime[0].split(':') : dateTime[1].split(':')
     const estimatedDuration = +time[1] + 60 * (+time[0] + +days * 24)
 
-    if (mission.missionStatus === MissionStatus.Ongoing || mission.missionStatus === MissionStatus.Paused) {
+    if (mission.endTime) {
```

Comment:
Have you tested that "if (mission.endTime)" behaves as expected without endTime?
I sometimes experienced that endTime is not null but instead some random time in the frontend

Reply:
It didn't use to work but with the change in Mission.cs allowing endTime to be either DateTimeOffset or null this works

Reply:
Nice